HLTrigger/Timer/plugins/FastTimerService.cc

```diff
@@ -1343,9 +1294,9 @@ void FastTimerService::printSummary(T& out, ResourcesPerJob const& data, std::st
   out << '\n';
   printPathSummaryHeader(out, "Processes and Paths");
   printSummaryLine(out, source.total, data.events, source_d.moduleLabel());
-  for (unsigned int i = 0; i < callgraph_.processes().size(); ++i) {
-    auto const& proc_d = callgraph_.processDescription(i);
-    auto const& proc = data.processes[i];
+  {
```

Comment:
Why the extra block ?

Reply:
And also above.

Reply:
`proc_d` and `proc` could just be defined only once and reused.

Reply:
I fixed this with your suggestion. That is better. Thanks.